client/layout/oauth-client/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const OauthClientLayout = ( { client } ) => (
+	<div className="oauth-client">
+		<nav>
+			<ul className="oauth-client__main-nav">
```

Comment:
I like how we're using semantic tags here. If we use flexbox styling here, I think we can simplify this component even further. I can pitch in with this if you'd like.